webview/src/webview_common.cpp

```diff
@@ -104,7 +104,11 @@ static int Create(lua_State* L)
     info.m_Self = dmScript::Ref(L, LUA_REGISTRYINDEX);
     info.m_L = dmScript::GetMainThread(L);
 
-    int webview_id = Platform_Create(L, &info);
+    // added by dotGears/TrungB
+    bool allowSiteAction = lua_toboolean(L, 2);
```

Comment:
A few things:
* When we add new options like this, we do it as part of a table. See our other api's for reference.
* The debug printf is left behind (printf doesn't even work on all platforms, so use dmLogInfo et al instead)